packages/maskbook/src/extension/options-page/DashboardDialogs/Wallet/Import.tsx

```diff
@@ -49,43 +51,28 @@ export function DashboardWalletImportDialog(props: WrappedDialogProps<object>) {
 
     const hdWallet = useWalletHD()
 
+    // wallet name
     const [name, setName] = useState('')
```

Comment:
```suggestion
    const [walletName, setWalletName] = useState('')
```

nit: I think we don't need comments here, let the variable names doc themself.